doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_mysql_sink.md

```diff
@@ -301,6 +308,14 @@ Despite the old encoding will be deprecated in the future, it is possible to swi
 
 [Top](#top)
 
+####<a name="section2.3.6"></a>About capping resources and expirating records
+Capping and expiration are disabled by default. Nevertheless, if desired, this can be enabled:
+
+* Capping by the number of records. This allows the resource growing up until certain configured maximum number of records is reached (`persistence_policy.max_records`), and then maintains a such a constant number of records.
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/pull/1366/commits/536811ecc9e7c7382eac81509a89ad3a34121fdf